### PR TITLE
Audit policy

### PR DESCRIPTION
**Pull request by mazzy89**
Add small paragraph about Audit Policy

### CODE REVIEW

**Comment by calvix** [comment]
maybe at least  show an example of a path with variables as version as `v_X_Y/files/....`

**Comment by mazzy89** [comment]
I know, unfortunately, we already have discussed this here https://gigantic.slack.com/archives/C7865GLSY/p1550655658010100 but this seems the most appropriate solution for the moment. Any proposals?

**Comment by mazzy89** [comment]
> maybe at least show an example of a path with variables as version as `v_X_Y/files/....`

Yes that it does not look a bad idea. I like that.

File: src/content/basics/kubernetes-on-giant-swarm/index.md [hunk 1]
```diff
@@ -42,6 +42,8 @@ As we are taking care of your cluster, we need to run some agents (e.g. for moni
 
 Similarly, some parts of the DNS, Ingress Controller, and Calico setups are visible to you inside your cluster. To ensure that your cluster runs without problems, please refrain from manipulating the `kube-system` namespace as well as the pods and other resources running in them if they are not documented.
 
+We customized the audit policy file by introducing rules that make the `Kube-apiserver` less verbose. The main philosophy behind is to mute those rules which have low-risk and produce high-volume of logs.  Check the manifest `audit-policy.yaml` in the repository [giantswarm/k8scloudconfig](https://github.com/giantswarm/k8scloudconfig) under `v_X_Y_Z/files/policies`.
```
Comment (marians) [inline src/content/basics/kubernetes-on-giant-swarm/index.md]:
```suggestion
We customize the audit policy file to eliminate rules which are both low-risk and produce a high volume of log entries. For full details, check the manifest `audit-policy.yaml` in the repository [giantswarm/k8scloudconfig](https://github.com/giantswarm/k8scloudconfig) under `v_X_Y_Z/files/policies`.
```

Suggesting some rephrasing

**Comment by marians** [comment]
I think this will work. Cool!

As a general note: Adding a single line break after each sentence in markup would simplify reviews and especially making suggestions to changes. It will also make future diffs more readable.

This is something we don't have consistently in our content, so it's absolutely not your fault.

File: src/content/basics/kubernetes-on-giant-swarm/index.md (hunk 1 above)
Comment (mazzy89) [reply src/content/basics/kubernetes-on-giant-swarm/index.md]: So we won't go with the convention `v_X_Y_Z` as proposed above by Vaclav?

Comment (marians) [reply src/content/basics/kubernetes-on-giant-swarm/index.md]: Yes, we should. I left that part untouched.

Comment (mazzy89) [reply src/content/basics/kubernetes-on-giant-swarm/index.md]: Oh right sorry. I was confused by GH

**Pull request merged**